### PR TITLE
add context

### PR DESCRIPTION
**Pull request by nmaro**

### CODE REVIEW

File: Etl.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ export class Etl {
     private _loaders: Loader[] = [];
     private _state: EtlState = EtlState.Stopped;
 
+    public constructor(private _context?: any) {}
```
Comment (mfeltscher) [inline Etl.ts]: Use `setContext()` to set the context inside the constructor.

**Comment by mfeltscher** [comment]
Also, add a changelog entry.

File: Etl.ts (hunk 1 above)
Comment (nmaro) [inline Etl.ts]: 4d19f4ee32c18b6bde64a5bed556d89758cf5601

**Comment by mfeltscher** [comment]
@dwirz LGTM, :shipit: ?

**Pull request merged**